pkg/sink/tui/misc/keybinds.go

```diff
@@ -145,7 +145,7 @@ func getPerProcCommandKeybindings() [][]string {
 func getContainerCommandKeybindings() [][]string {
 	return [][]string{
 		{"Quit: q or <C-c>"},
-		{"Pause Rendering: s"},
+		{"Pause Rendering: p"},
```

Comment:
Is there a reason for changing `s` to `p` for pause?

Reply:
Yes, "p" for pause :P